src/test/java/teammates/test/cases/pagedata/InstructorCourseStudentDetailsPageDataTest.java

```diff
@@ -107,10 +115,10 @@ private void createStudentProfile(String email, String pictureKey) {
         } else {
             this.pictureUrl = Const.ActionURIs.STUDENT_PROFILE_PICTURE + "?"
                             + Const.ParamsNames.BLOB_KEY + "=" + pictureKey + "&"
-                            + Const.ParamsNames.USER_ID + "=null";
+                            + Const.ParamsNames.USER_ID + "=" + GOOGLE_ID;
```

Comment:
I think you misunderstand the meaning of the `USER_ID` parameter and added `GOOGLE_ID` wrongly in this whole file. It is used to indicate the Google ID of the person requesting the page (instructor in this case). However here you seem to interpret it as the Google ID of the student.